consensus/polybft/checkpoint_manager.go

```diff
@@ -5,7 +5,7 @@ import (
 	"math/big"
 	"strconv"
 
-	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	bls "github.com/0xPolygon/bnsnark1/core"
```

Comment:
Is it worth to rename core to bls? There is an alias for every import.

Reply:
I am not sure :( And I am not sure that bls is good name. Because bn and bls are different curves. I kept bls as import name just to prevent too much changes in original files